rele/worker.py

```diff
@@ -134,7 +134,13 @@ def create_and_run(subs, config):
         config.threads_per_subscription,
     )
 
-    signal.signal(signal.SIGINT, signal.SIG_IGN)
+    # to allow killing runrele worker via ctrl+c
+    signal.signal(signal.SIGINT, worker.stop)
     signal.signal(signal.SIGTERM, worker.stop)
-    signal.signal(signal.SIGTSTP, worker.stop)
+    try:
+        signal.signal(signal.SIGTSTP, worker.stop)
+    except:
+        # SIGSTP doesn't exist on windows, so we use SIGBREAK instead
+        signal.signal(signal.SIGBREAK, worker.stop)
```

Comment:
Would it be better to check if you are on windows here and use Sigbreak directly? 
Im thinking something like:
```suggestion
    if sys.platform.startswith('win'):
        # SIGSTP doesn't exist on windows, so we use SIGBREAK instead
        signal.signal(signal.SIGBREAK, worker.stop)
    else:
    	signal.signal(signal.SIGTSTP, worker.stop)
```

Reply:
In addition to @andrewgy8 suggestion, what do you think about extracting the signal stop retrieval to keep the method cleaner?

Something like this:
```
def _get_stop_signal():
    if sys.platform.startswith('win'):
	        # SIGSTP doesn't exist on windows, so we use SIGBREAK instead
	        return signal.SIGBREAK

    return signal.SIGTSTP
```

Then inside `create_and_run`:

```
    signal.signal(self._get_stop_signal(), worker.stop)
```